.github/CODEOWNERS

```diff
@@ -22,6 +22,7 @@
 
 cmd/configschema/                                        @open-telemetry/collector-contrib-approvers @mx-psi @dmitryax @pmcollins
 cmd/mdatagen/                                            @open-telemetry/collector-contrib-approvers @dmitryax
+cmd/opampsupervisor/                                     @open-telemetry/collector-contrib-approvers @evan-bradley @atoulme @tigrannajaryan
```

Comment:
Are any of these teams appropriate code owners? https://github.com/orgs/open-telemetry/teams?query=opamp